app/models/concerns/sufia/solr_document_behavior.rb

```diff
@@ -50,6 +50,16 @@ def date_uploaded
       end
     end
 
+    def create_date
+      field = self[Solrizer.solr_name("system_create", :stored_sortable, type: :date)]
+      return unless field.present?
+      begin
+        Date.parse(field).to_formatted_s(:standard)
+      rescue
```

Comment:
It might be better to rescue from the specific data parsing error, instead of just any error.